src/frontend/src/pages/CreditsPage/CreditsPage.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Grid, Button, Typography } from '@mui/material'; // some imports from libraries
+import { useState } from 'react';
+import PageTitle from '../../layouts/PageTitle/PageTitle'; // ...and from other files
+import PageBlock from '../../layouts/PageBlock';
+
+const CreditsPage: React.FC = () => {
+  const names: {
+    name: string;
+    color: string;
+  }[] = [{ name: 'Kevin Chen', color: '#00A36C' }]; // This is a list of names, add yours here! (along with a hexcode color)
```

Comment:
can you explain that this is a javascript object, that the names are keys and and the name/hex are values